docs/src/routes/docs/components/dialog.svx

```diff
@@ -0,0 +1,101 @@
+---
+name: Dialog
+shadowRaised: '0 2px 4px -1px transparentize(black, .8), 0 3px 5px 0 transparentize(black, .86), 0 1px 10px 0 transparentize(black, .88);'
```

Comment:
This should have actual values, as it won't undergo any preprocessing. Copy it from https://github.com/illright/attractions/blob/develop/docs/src/routes/docs/components/button.svx#L4